pulsar-broker/src/main/java/org/apache/pulsar/broker/PulsarService.java

```diff
@@ -902,9 +902,8 @@ public void start() throws PulsarServerException {
                     + (config.getWebServicePortTls().isPresent() ? ", tls-port = " + config.getWebServicePortTls() : "")
                     + (StringUtils.isNotEmpty(brokerServiceUrl) ? ", broker url= " + brokerServiceUrl : "")
                     + (StringUtils.isNotEmpty(brokerServiceUrlTls) ? ", broker tls url= " + brokerServiceUrlTls : "");
-            LOG.info("messaging service is ready, bootstrap_seconds={}", bootstrapTimeSeconds);
-            LOG.info("messaging service is ready, {}, cluster={}, configs={}", bootstrapMessage,
-                    config.getClusterName(), config);
+            LOG.info("messaging service is ready, {}, cluster={}, configs={}, bootstrap_seconds={}", bootstrapMessage,
```

Comment:
It's better move the `bootstrap_seconds` to  first position,  because the  `configs` will be very long

Reply:
+1

Reply:
@codelipenghui @AnonHxy I have updated, PTAL.